python/pythondotorg
# Admin-editable dynamic content streams

#### What's this PR do?

Adds support for dynamic content sections which are powered by RSS feeds.

The combination of these feeds is admin-controllable.
#### Where should the reviewer start?

`models.py`, then likely the changes to the `update_blog` management command, followed by the template tag.
#### How should this be manually tested?

Add a couple of feeds to the automatically created `psf-news` `FeedAggregate`. Run `./manage.py update_blogs`. You should see the contents of that feed at http://localhost:8000/psf-landing/ in date descending order.
#### What are the relevant tickets?

Fixes #476
#### Screenshots (if appropriate)

![screen shot 2014-12-13 at 1 11 32 pm](https://cloud.githubusercontent.com/assets/3853/5425390/95e5ca32-82c9-11e4-9010-0002b1d6d4a9.png)
#### What gif best describes this PR or how it makes you feel?

![](http://s.mlkshk-cdn.com/r/10XHJ.jpg)

(this is loosely formatted based on a [PR template that I use at work](http://quickleft.com/blog/pull-request-templates-make-code-review-easier))
